src/screens/FeedScreen/Content.js

```diff
@@ -72,6 +72,10 @@ const Content = ({
     message
   );
 
+  const handleHeightContainer = ({nativeEvent}) => {
+    setLayoutHeight(nativeEvent.layout.height);
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new function `handleHeightContainer` has been added to manage layout height changes. This is a good practice as it allows the component to adapt to different content sizes dynamically.
```diff
+ const handleHeightContainer = ({nativeEvent}) => {
+   setLayoutHeight(nativeEvent.layout.height);
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->